data/preprocess.py

```diff
@@ -18,7 +18,8 @@ def pull_data(output: str, connection: models.Connection) -> None:
 
         owner_writer = csv.DictWriter(
             owner_file,
-            fieldnames=['domain', 'organization_en', 'organization_fr']
+            fieldnames=['domain', 'organization_en', 'organization_fr'],
+            extrasaction='ignore'
         )
         domain_writer = csv.DictWriter(domain_file, fieldnames=['domain'])
```

Comment:
Shouldn't tthis line, and the one below it, also have this argument added for the same reason?

Reply:
Yup. I'll do that.